sdk/communication/Azure.Communication.MediaComposition/tests/samples/Sample1_MediaCompositionClient.cs

```diff
@@ -95,7 +94,7 @@ public async Task UpsertInputsMediaCompositionAsync()
                 ["james"] = new ParticipantInput
                 (
                     id: new MicrosoftTeamsUserIdentifier("f3ba9014-6dca-4456-8ec0-fa03cfa2b70p"),
```

Comment:
This should be CommunicationUser if it is coming from ACS Group call

Reply:
switched to CommunicationUser. Pushed new changes.